diff --git a/node_modules/@shopify/react-native-skia/android/CMakeLists.txt b/node_modules/@shopify/react-native-skia/android/CMakeLists.txt
index f48752a..18de9ad 100644
--- a/node_modules/@shopify/react-native-skia/android/CMakeLists.txt
+++ b/node_modules/@shopify/react-native-skia/android/CMakeLists.txt
@@ -11,6 +11,15 @@ set (SKIA_SVG_LIB "svg")
 set (SKIA_SKSHAPER_LIB "skshaper")
 set (SKIA_SKPARAGRAPH_LIB "skparagraph")
 set (SKIA_SKUNICODE_CORE_LIB "skunicode_core")
+
+if (CMAKE_LIBRARY_OUTPUT_DIRECTORY)
+    file(MAKE_DIRECTORY "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
+endif()
+
+if (CMAKE_RUNTIME_OUTPUT_DIRECTORY)
+    file(MAKE_DIRECTORY "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}")
+endif()
+
 set (SKIA_SKUNICODE_ICU_LIB "skunicode_icu")
 
 
